docs/src/prototypes/dropdowns/inputWithDropdown.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import * as _ from 'lodash'
+import keyboardKey from 'keyboard-key'
+import { Provider, Dropdown, DropdownProps, Input, themes } from '@stardust-ui/react'
+
+import { atMentionItems, AtMention } from './dataMocks'
+import { insertNodeAtCursorPosition, removeElement } from './utils'
+
+interface InputWithDropdownState {
+  dropdownValue?: string
+  searchQuery?: string
+}
+
+class InputWithDropdownExample extends React.Component<{}, InputWithDropdownState> {
+  private readonly mountNodeId = 'dropdown-mount-node'
+  private readonly dropdownInputSelector = `#${this.mountNodeId} .${Input.slotClassNames.input}`
+  private readonly editorStyle: React.CSSProperties = {
+    backgroundColor: 'lightgrey',
+    padding: '5px',
+    minHeight: '100px',
+    outline: 0,
+  }
+
+  private dropdownExists = false
+  private contendEditableRef = React.createRef<HTMLDivElement>()
+
+  public state: InputWithDropdownState = {
+    dropdownValue: null,
+    searchQuery: '',
+  }
+
+  render() {
+    return (
+      <div
+        contentEditable
+        ref={this.contendEditableRef}
+        onKeyUp={this.handleEditorKeyUp}
+        style={this.editorStyle}
+      />
+    )
+  }
+
+  private showDropdown = () => {
+    this.dropdownExists = true
+    insertNodeAtCursorPosition({ id: this.mountNodeId })
+
+    const node = this.getMountNode()
+    ReactDOM.render(
```

Comment:
https://github.com/stardust-ui/react/pull/901#discussion_r256901115 from @layershifter 

Can you please why we can't use `createPortal()`/`PortalInner` component there? What is a reason for another React's root?

Reply:
no reason, 1st iteration, let me try that, thanks for the suggestion

Reply:
implemented using `createPortal`, works as expected, good call